src/System.Collections/src/System/Collections/Generic/SortedSet.TreeSubSet.cs

```diff
@@ -45,12 +45,6 @@ public TreeSubSet(SortedSet<T> Underlying, T Min, T Max, bool lowerBoundActive,
                 VersionCheckImpl();
             }
 
-            private TreeSubSet(SerializationInfo info, StreamingContext context)
```

Comment:
How do you know from your analysis whether the SortedSets people were serializing were TreeSubSet or not?  We could start by not having them and only later opting them in if really important, it's just very difficult to reason about.  "You can serialize SortedSet... but only if you didn't get it from these APIs on SortedSet."